vsintegration/src/unittests/Tests.LanguageService.QuickInfo.fs

```diff
@@ -919,7 +919,7 @@ type QuickInfoTests() =
                 type A() =
                     static member Foo([<System.ParamArrayAttribute>] a : int[]) = ()
                 let r = A.Foo(42)""" ,
-            "type A","params a:"    )
+            "type A","[<ParamArray>] a:"    )
```

Comment:
On types, it's always printed in the short form https://github.com/Microsoft/visualfsharp/blob/742c13304ce98a0af84bfcd5bf20afdbe3a74cdb/src/fsharp/vs/ServiceDeclarations.fs#L809. The check is at https://github.com/dungpa/visualfsharp/blob/93bd0997069713d8f0fed10d3af794bf66928e1c/src/fsharp/NicePrint.fs#L87-L88. It seems ok because this is displayed by language service, not FSI response and signature generation.
What do you think?